Formula/picard-tools.rb

```diff
@@ -0,0 +1,20 @@
+class PicardTools < Formula
+  desc "Tools for manipulating HTS data and formats"
+  homepage "https://broadinstitute.github.io/picard/"
+  url "https://github.com/broadinstitute/picard/releases/download/2.16.0/picard.jar"
+  sha256 "01cf3c930d2b4841960497491512d327bf669c1ed2e753152e1e651a27288c2d"
+
+  bottle :unneeded
+
+  depends_on :java
+
+  def install
+    java = share/"java"
+    java.install Dir["*.jar"]
+    bin.write_jar_script java/"picard.jar", "picard"
+  end
+
+  test do
+    assert_match "USAGE", shell_output("#{bin}/picard -h 2>&1", 1)
```

Comment:
We need a test that does more than output usage info

Reply:
I've added a test.